docs/docsite/rst/vmware/scenario_clone_template.rst

```diff
@@ -0,0 +1,124 @@
+        - Virtual machine.Provisioning.Read customization specifications on the root vCenter Server if you are customizing the guest operating system.
+
+Assumptions
+===========
+
+- All variable names and VMware object names are case sensitive
+- VMware allows creation of virtual machine and templates with same name across datacenters and within datacenters.
+- You need to use Python 2.7.9 version in order to use *validate_certs* option, as this version is capable of changing the SSL verification behaviours.
+
+Caveats
+=======
+
+- Hosts in the ESXI cluster must have access to the datastore that the template resides on.
+- Multiple templates with the same name will cause module failures.
+- In order to utilize Guest Customization, VMWare Tools must be installed on the template. For Linux, the open-vm-tools package is recommended, and it requires that Perl be installed.
+
+
+Example Description
+===================
+
+In this use case / example, we will be selecting a virtual machine template and cloning it into a specific folder in our Datacenter / Cluster.  The following Ansible playbook showcases the basic parameters that are needed for this.
+
+.. code-block:: yaml
+
+    ---
+    - name: Create a VM from a template
+      hosts: localhost
+      connection: local
+      gather_facts: no
+      tasks:
+      - name: Clone the template
+        vmware_guest:
+          hostname: 192.0.2.44
+          username: administrator@vsphere.local
+          password: vmware
+          validate_certs: False
+          name: testvm_2
+          template: template_el7
+          datacenter: dc1
+          folder: /dc1/vm
+          state: poweredon
+          wait_for_ip_address: yes
+
+
+Since Ansible utilizes the VMware API to perform actions, in this use case we will be connecting directly to the API from our localhost. This means that our playbooks will not be running from the vCenter or ESXi Server. We do not necessarily need to collect facts about our localhost, so the *gather_facts* parameter will be disabled. You can run these modules against another server that would then connect to the API if your localhost does not have access to vCenter. If so, the required Python modules will need to be installed on that target server.
+
+To begin, there are a few bits of information we will need. First and foremost is the hostname of the ESXi server or vCenter server. After this, we will need the username and password for this server. For now, you will be entering these directly, but in a more advanced playbook this can be abstracted out and stored in a more secure fashion [1][2]. If your vCenter or ESXi server is not setup with proper CA certificates that can be verified from the Ansible server, then it is necessary to disable validation of these certificates by using the *validate_certs* parameter. To do this you need to set `validate_certs=False` in your playbook.
```

Comment:
rST uses two backticks for `code quotations`, so you need to add another set around `validate_certs=False`